settings.yml

```diff
@@ -0,0 +1,11 @@
+---
+project: fwk_core
+title: XCORE Core Framework
+version: 1.0.2
```

Comment:
I find it disappointing that the `title`, `project` and `version` key-value pairs have been duplicated here and in settings.json.  It may be unavoidable, but I cannot tell you the number of times an unwary engineer has updated one version string and forgotten to update the other one.

Reply:
See below comment by @lucianomartin  - the json file is now obsolete